programs/voter-stake-registry/tests/program_test/governance.rs

```diff
@@ -282,16 +301,16 @@ impl GovernanceRealmCookie {
                 proposal::VoteType::SingleChoice,
                 vec!["yes".into()],
                 true,
-                0,
-            ),
-            spl_governance::instruction::add_signatory(
-                &self.governance.program_id,
-                &proposal,
-                &voter.token_owner_record,
-                &authority.pubkey(),
-                &payer.pubkey(),
-                &authority.pubkey(),
+                &proposal_seed,
             ),
+            // spl_governance::instruction::add_signatory(
+            //     &self.governance.program_id,
+            //     &proposal,
+            //     &voter.token_owner_record,
+            //     &authority.pubkey(),
+            //     &payer.pubkey(),
+            //     &authority.pubkey(),
+            // ),
```

Comment:
how come this is no longer needed? in general we prefer not to keep unused code in comments

Reply:
sorry I missed this, the add_signatory() has changed in spl-gov, newer tests did not have it when creating a proposal.

I readded it with the new version of the instruction.